src/Notepad2.rc

```diff
@@ -335,6 +335,14 @@ BEGIN
 			MENUITEM "Esca&pe C Chars\tCtrl+Alt+E",				IDM_EDIT_ESCAPECCHARS
 			MENUITEM "&Unescape C Chars\tCtrl+Alt+R",			IDM_EDIT_UNESCAPECCHARS
 			MENUITEM SEPARATOR
+			POPUP "Base64"
+			BEGIN
+				MENUITEM "Standard Encode\t",							IDM_EDIT_BASE64_ENCODE

```

Comment:
`\t` should be removed for menu without shortcut.

Reply:
You are right, sorry. I had put them in temporarily but then I forgot them